docs/style.css

```diff
@@ -273,7 +273,7 @@ footer p + p:before {
 .postlight-link {
   overflow: hidden;
   width: 6.3888888889rem; /* 115 / 27 */
```

Comment:
This value can probably change too. It looks like the new asset's 100 units wide by 20 units tall, so the width could be 1.3 * 100 / 20 = 6.5rem